guides/source/upgrading_ruby_on_rails.md

```diff
@@ -169,93 +169,109 @@ more meaningful preview images. Previously the first frame of the video would be
 and that caused problems if the video faded in from black. This change requires
 FFmpeg v3.4+.
 
-### ActiveStorage variant processor changed to :vips
+### Active Storage default variant processor changed to `:vips`
 
-Image transformation will now use libvips instead of ImageMagick. This will reduce
+For new apps, image transformation will use libvips instead of ImageMagick. This will reduce
 the time taken to generate variants as well as CPU and memory usage, improving response
 times in apps that rely on active storage to serve their images. 
 
 The `:mini_magick` option is not being deprecated, so it is fine to keep using it.
 
-Migrating to libvips requires changing existing image transformation code to the
-`image_processing` macros, and replacing ImageMagick's options with libvips' options.
+To migrate an existing app to libvips, set:
+
+```ruby
+Rails.application.config.active_storage.variant_processor = :vips
+```
+
+You will then need to change existing image transformation code to the
+`image_processing` macros, and replace ImageMagick's options with libvips' options.
 
 #### Replace resize with resize_to_limit
 ```diff
 - variant(resize: "100x")
 + variant(resize_to_limit: [100, nil])
 ```
 
-If you forget to do this, when you switch to vips you will see this error: `no implicit conversion to float from string`.
+If you don't do this, when you switch to vips you will see this error: `no implicit conversion to float from string`.
 
 #### Use an array when cropping
 ```diff
 - variant(crop: "1920x1080+0+0")
 + variant(crop: [0, 0, 1920, 1080])
 ```
 
-If you forget to do this, when you switch to vips you will see this error: `unable to call crop: you supplied 2 arguments, but operation needs 5`.
+If you don't do this, when you switch to vips you will see this error: `unable to call crop: you supplied 2 arguments, but operation needs 5`.
```

Comment:
I think moving the comma makes more sense, as "not doing this" only applies to moving to vips.
I also think having `this` refer to something later in the sentence is a bit odd.
```suggestion
If you don't do this when migrating to vips, you will see the following error: `unable to call crop: you supplied 2 arguments, but operation needs 5`.
```